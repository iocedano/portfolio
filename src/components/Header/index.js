import React from 'react';
import logo from "../../assets/logo.svg";
import "./styled.css";

const Header = () => (
  <header className="header-container">
    <img src={logo} className="header-logo" alt="logo" />
    <p>
      Edit <code>src/App.tsx</code> and save to reload.
    </p>
    <a
      className="header-link"
      href="https://reactjs.org"
      target="_blank"
      rel="noopener noreferrer"
    >
      Learn React
    </a>
  </header>
);

export default Header;
