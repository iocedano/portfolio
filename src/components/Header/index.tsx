import React from 'react';
import "./styled.css";

const Header = () => (
  <header className="header-container">
    <nav className="nav-container">
      <ul className="nav-list">
        <li className="nav-item">Job Histories</li>
        <li className="nav-item">Skills</li>
        <li className="nav-item">Contact</li>
      </ul>
    </nav>
  </header>
);

export default Header;
