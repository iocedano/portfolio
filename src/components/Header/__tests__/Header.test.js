import React from "react";
import { render } from "@testing-library/react";
import Header from "..";

describe("Header", () => {
  it("should render", () => {
    const { container } = render(<Header />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
