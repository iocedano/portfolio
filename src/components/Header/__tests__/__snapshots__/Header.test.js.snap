// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render 1`] = `
<header
  class="header-container"
>
  <nav
    class="nav-container"
  >
    <ul
      class="nav-list"
    >
      <li
        class="nav-item"
      >
        Job Histories
      </li>
      <li
        class="nav-item"
      >
        Skills
      </li>
      <li
        class="nav-item"
      >
        Contact
      </li>
    </ul>
  </nav>
</header>
`;
